### PR TITLE
Adding DSC lead to the map

### PR DESCRIPTION
**Pull request by Manjuphoenix**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@Manjuphoenix Please redo your changes. You should only use the skills mentioned. Also follow the same formatting pattern. Don't add extra quotes or change anything else in formatting. You're missing quotes and comma as well. And change the image to `jpg`.

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -245,6 +246,21 @@ const data = [
 		linkedin: "https://linkedin.com/in/sshthassanraza",
 		twitter: "https://twitter.com/sshthassanraza"
 	},
+	{
+        "latitude": 13.039039,
+        "longitude": 77.493095,
+        "batch": 2020,
+        "photo": Manjunath,
+        "region": "Asia",
+        "skills": ["AI/ML", App Dev", "Web Dev", "Devops", "Cloud"],
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:255]: Please fix this line as well.

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me now.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@Manjuphoenix Please delete `src/photos/Manjunath.jpeg` as well.

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "region": "Asia",
+        "skills": ["AI/ML", "App Dev", "Web Dev", "Cloud"],
+        "name": "Manjunath",
+        "bio": "In a journey of making a dent in the field of Science and Technology",
+        "url": "https://dsc.community.dev/atria-institute-of-technology/",
+        "chapterName": "Atria Institute of Technology",
+        "github": "https://github.com/Manjuphoenix",
+        "linkedin": "https://linkedin.com/in/manjunath-d",
+        "twitter": "https://twitter.com/manjumic"
+	}
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:263]:
```suggestion
	},
```

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]

File: src/components/json/worldData.js [hunk 3]
```diff
@@ -245,6 +246,21 @@ const data = [
 		linkedin: "https://linkedin.com/in/sshthassanraza",
 		twitter: "https://twitter.com/sshthassanraza"
 	},
+	{
+        "latitude": 13.039039,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:250]:
```suggestion
        latitude: 13.039039,
```

File: src/components/json/worldData.js [hunk 4]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "longitude": 77.493095,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:251]:
```suggestion
        longitude: 77.493095,
```

File: src/components/json/worldData.js [hunk 5]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "batch": 2020,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:252]:
```suggestion
        batch: 2020,
```

File: src/components/json/worldData.js [hunk 6]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "photo": Manjunath,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:253]:
```suggestion
        photo: Manjunath,
```

File: src/components/json/worldData.js [hunk 7]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "region": "Asia",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:254]:
```suggestion
        region: "Asia",
```

File: src/components/json/worldData.js [hunk 8]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "skills": ["AI/ML", "App Dev", "Web Dev", "Cloud"],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:255]:
```suggestion
        skills: ["AI/ML", "App Dev", "Web Dev", "Cloud"],
```

File: src/components/json/worldData.js [hunk 9]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "name": "Manjunath",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:256]:
```suggestion
        name: "Manjunath",
```

File: src/components/json/worldData.js [hunk 10]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "bio": "In a journey of making a dent in the field of Science and Technology",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:257]:
```suggestion
        bio: "In a journey of making a dent in the field of Science and Technology",
```

File: src/components/json/worldData.js [hunk 11]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "url": "https://dsc.community.dev/atria-institute-of-technology/",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:258]:
```suggestion
        url: "https://dsc.community.dev/atria-institute-of-technology",
```

File: src/components/json/worldData.js [hunk 12]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "chapterName": "Atria Institute of Technology",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:259]:
```suggestion
        chapterName: "Atria Institute of Technology",
```

File: src/components/json/worldData.js [hunk 13]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "github": "https://github.com/Manjuphoenix",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:260]:
```suggestion
        github: "https://github.com/Manjuphoenix",
```

File: src/components/json/worldData.js [hunk 14]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "linkedin": "https://linkedin.com/in/manjunath-d",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:261]:
```suggestion
        linkedin: "https://linkedin.com/in/manjunath-d",
```

File: src/components/json/worldData.js [hunk 15]
```diff
@@ -245,6 +246,21 @@ const data = [
...
+        "twitter": "https://twitter.com/manjumic"
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:262]:
```suggestion
        twitter: "https://twitter.com/manjumic"
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Pull request merged**